### ISSUE TITLE
[PolishTracker] Using cookies method (instead of login/password)

### ISSUE THREAD

**Issue by sevospl** [Jackett/Jackett#8780]
PolishTracker doesn't allow multiple sessions, so the only way to get around it would be using the same cookies from the browser (just like it's done with some trackers), otherwise either Jackett or the browser gets logged out. Using the same cookies would solve this issue.

**Comment by ngosang** [comment]
I think it's fine as it is. The cookie method has problems too, it expires each few days and you have to reconfigure Jackett. If this is a huge problem for you talk to the admin/staff of PolishTracker to allow 2 sessions at the same time.

**Comment by sevospl** [comment]
@ngosang They simply don't (and won't) allow logging from multiple devices. And the only way to get around that is having the same cookies. They don't expire each few days but each 3 months which is more than enough and shouldn't cause issues. Therefore I would appreciate the cookies method implementation.

**Comment by ngosang** [comment]
> They simply don't (and won't) allow logging in from multiple devices.

Did you talk to the admins? Explain this issue to them, It's reasonable to allow multiple devices. If they are not interested paste the response and I will do the changes in the code.

**Comment by sevospl** [comment]
Hey @ngosang. Yes, I did. Because of how their back-end is designed, it's impossible to allow multiple sessions. They advised to use cookies-based method authorization.

**Comment by RoloSoze** [comment]
@ngosang Do you plan on switching PolishTracker to the cookie method to allow multiple sessions?

**Comment by garfield69** [comment]
Jackett 0.16.1523

**Comment by sevospl** [comment]
Hi @garfield69 and @ngosang.

I would love to let you know that PolishTrackers' API has been updated. It now can be fully used by Jackett. I additionaly asked their staff to make sure it's ready to use.

As a user [you can get an API key](https://pte.nu/user/api) and [API is well documented](https://cdn.pte.nu/docs/index.html).

TL;DR: there are 2 endpoints: [Torrents List](https://cdn.pte.nu/docs/index.html#torrents-list) and [Torrent Download](https://cdn.pte.nu/docs/index.html#torrent-download).

This means there is no longer need for cookie method. API would be the preferred way to go and I hope someone would like to take it and implement API method :)
